tree/dataframe/src/RDFInterfaceUtils.cxx

```diff
@@ -92,7 +92,7 @@ struct ParsedExpression {
 };
 
 /// Look at expression `expr` and return a pair of (column names used, aliases used)
-static std::pair<ColumnNames_t, ColumnNames_t>
+std::pair<ColumnNames_t, ColumnNames_t>
```

Comment:
here and below, the extra `static` was something that @pcanal or @Axel-Naumann suggested at some point as the way we write this in ROOT -- I don't remember exactly why, I think it had to do with MSVC++?

I'm ok with removing it but let's ping them to be sure.

Reply:
For Windows: on the contrary, static members are problematic across DLLs.

Could have been because of cling? I really don't know, `git blame` might be helpful here!

If the CI is happy then this looks like a good change to me.

Reply:
Thanks for the discussion! The CI is happy, so let's try to remove the `static` in these situations, like `clang-tidy` suggests.